nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/trash/test/TestTrashBackwardMechanism.java

```diff
@@ -120,12 +116,10 @@ public void testMechanismTriggersOneListener() {
 
     public static class BulkListenersFilter implements LogCaptureFeature.Filter {
 
-        protected final Set<String> bulkListenerNames = Stream.of(BulkLifeCycleChangeListener.class,
-                BulkTrashedStateChangeListener.class).map(Class::getName).collect(Collectors.toSet());
-
         @Override
         public boolean accept(LoggingEvent event) {
-            return Level.DEBUG.equals(event.getLevel()) && bulkListenerNames.contains(event.getLoggerName());
+            return Level.DEBUG.equals(event.getLevel())
```

Comment:
Test principle seems to have change regarding the comment on the test method and the test method name.
This test was there to ensure we don't loop twice and also to prevent issues on MySQL/MariaDB where transaction level could generate issues depending on which listener was called first.

Reply:
Actually, with MySQL the tests fails without the Filter and works fine with, what's the problem ?